pkg/jx/cmd/create_env_test.go

```diff
@@ -133,15 +133,15 @@ func TestCreateEnvRun(t *testing.T) {
 	helmValuesConfig.ExposeController.Config.HTTP = "false"
 	helmValuesConfig.ExposeController.Config.TLSAcme = "false"
 
+	commonOpts := cmd.NewCommonOptionsWithFactory(factory)
+	commonOpts.In = console.In
+	commonOpts.Out = console.Out
+	commonOpts.Err = console.Err
```

Comment:
I wonder if we should do a helper func for this as it's quiet a common pattern in tests - we can add that later!

Reply:
Agree. Let's add it as a follow up.